tests/mocha/lists_test.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
```

Comment:
```suggestion
 * Copyright 2022 Google LLC
```

Also, could you give this file a more descriptive name eg `list_blocks_test`?